docs/developer/template_switches.asciidoc

```diff
@@ -288,6 +288,61 @@ So here is mini templating language to format values.
 ** switch._wsUser
 ** vlan
 
+* bounce
```

Comment:
I was able to use `$last_accounting.acctsessionid` in `bounce` scope but it is not referenced here.

Some vendors require to use `Acct-Session-Id` attribute to identify port to bounce.